File: wathermelon/awnser.py
import sys


def can_split_evenly(weight: int) -> bool:
    return weight % 2 == 0 and weight >= 4


def main() -> None:
    data = sys.stdin.read().strip().split()
    if not data:
        return

    weight = int(data[0])

    if can_split_evenly(weight):
        print("YES")
    else:
        print("NO")


if __name__ == "__main__":
    main()

File: wathermelon/awnser.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func canSplitEvenly(weight int) bool {
	return weight%2 == 0 && weight >= 4
}

func main() {
	inputReader := bufio.NewReader(os.Stdin)
	outputWriter := bufio.NewWriter(os.Stdout)
	defer outputWriter.Flush()

	var weight int
	if _, err := fmt.Fscan(inputReader, &weight); err != nil {
		return
	}

	if canSplitEvenly(weight) {
		fmt.Fprintln(outputWriter, "YES")
	} else {
		fmt.Fprintln(outputWriter, "NO")
	}
}
